mosec/mixin/numbin_worker.py

```diff
@@ -40,16 +36,20 @@ class NumBinIPCMixin:
 
     def serialize_ipc(self, data: Any) -> bytes:
         """Serialize with NumBin for the IPC."""
+        import numbin
```

Comment:
May I know the benefit of using import here instead of the head of the file?

Reply:
The previous implementation requires installing all the optional dependencies when you `from mosec.mixin import XXX`. Although the ImportError is suppressed by warnings.